### PR TITLE
Updating Snackbar documentation

### PR DESCRIPTION
**Pull request by juancarlosjr97**
Adding to the Snackbar documentation on how to change the color of the action button of a snackbar.

### Summary

I was trying to change the color of the action button on a snackbar. Chatting online with one the developer, it is passing the theme props, changing the `accent` color.

### CODE REVIEW

File: src/components/Snackbar.tsx [hunk 1]
```diff
@@ -270,6 +270,9 @@ class Snackbar extends React.Component<Props, State> {
                 onDismiss();
               }}
               style={styles.button}
+                /**
+                * Change the color of the text of the action of the snackbar passing a theme prop.
```
Comment (Trancever) [inline src/components/Snackbar.tsx:274]: This shouldn't be done here. Above the component, we have a commented section that provides an example of the component usage, and we can also add additional information there.

**Review by Trancever — CHANGES_REQUESTED** [review]

File: src/components/Snackbar.tsx (hunk 1 above)
Comment (juancarlosjr97) [reply src/components/Snackbar.tsx:274]: Sorry, where above @Trancever ?? Can you tell so I can make the change, please.

Comment (Trancever) [reply src/components/Snackbar.tsx:274]: Check line 60

**Comment by Trancever** [comment]
@juancarlosjr97 Please reopen the PR if it is still relevant. I am closing because of inactivity and conflicts.

**Pull request closed without merging**